ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/ReadOnlySqlLedger.scala

```diff
@@ -32,38 +27,30 @@ import scala.concurrent.{ExecutionContext, Future}
 
 object ReadOnlySqlLedger {
 
-  val noOfShortLivedConnections = 16
+  val maxConnections = 16
 
   //jdbcUrl must have the user/password encoded in form of: "jdbc:postgresql://localhost/test?user=fred&password=secret"
   def apply(
       jdbcUrl: String,
       ledgerId: Option[LedgerId],
       loggerFactory: NamedLoggerFactory,
       metrics: MetricRegistry,
-  )(implicit mat: Materializer): Future[ReadOnlyLedger] = {
-    implicit val ec: ExecutionContext = DEC
-
+  )(implicit mat: Materializer): Resource[ReadOnlyLedger] = {
+    implicit val ec: ExecutionContext = mat.executionContext
```

Comment:
Why changing the execution context?

Reply:
Makes no difference and it's one fewer edge case.